src/Ml.Cli.WebApp/ClientApp/src/Server/Project/Page/Page.container.js

```diff
@@ -74,22 +95,37 @@ export const initialState = {
     numberAnnotationsDone: 0,
     numberAnnotationsToDo:0,
     percentageNumberAnnotationsDone:0
-  }
+  },
+  isModalOpened: false
 };
 
 const usePage = (fetch) => {
   const { id } = useParams();
   const [state, dispatch] = useReducer(reducer, initialState);
   const onExport = projectId => fetchExportAnnotations(fetch)(projectId);
+  const lock = {
+    onCancel: () => dispatch({type: 'open_modal', data: {isModalOpened: false}}),
+    onSubmit: async () => {
+      const response = await fetchDeleteProject(fetch)(id);
+      let data;
+      if (response.status >= 500) {
```

Comment:
**suggestion**: 500 is a magic number => constant

Reply:
Done